projects/js-packages/publicize-components/src/hooks/use-anaytics-tracks/README.md

```diff
@@ -0,0 +1,24 @@
+# useAnalyticsTracks hook
+This hook is used to add some functionality around tracks analytics
+Handles initializing the user and allows for easy setup of callbacks
+This implementation only records events when the current user has a Jetpack connection
+
+```es6
+
+const Component = () => {
+	const { recordEventHandler, recordEvent } = useAnalyticsTracks( {
+			pageViewEventName: 'view_event_name',
+			pageViewNamespace: 'jetpack',
+			pageViewSuffix: '',
+		} );
+	const recordReviewClick = recordEventHandler( 'event_name', {} );
```

Comment:
IMO, `recordEventHandler` is redundant and can be simply recreated by using:
`useCallback( () => recordEvent( 'event_name', {} ), [] )`